### PR TITLE
chore(website): use an svg to animate the triangles

### PR DESCRIPTION
**Pull request by coldpour**
### Description
remove the triangles javascript functions and add a static svg file with the same hotness

### Motivation and context

laptop fanspeed
connect #429

### Screenshots, videos, or demo, if appropriate

https://429-polish-triangles--mineral-ui.netlify.com/

### How to test

1. `cd mineral-ui && npm start`
2. open the home page and scroll to the bottom

### Types of changes

Chore

### Checklist

* [ ] Renders and functions properly in [all supported browsers](https://github.com/mineral-ui/mineral-ui#browser-support)
* [ ] Automated tests written and passing
* [ ] [Accessibility](http://webaim.org/intro) and [inclusivity](https://24ways.org/2016/what-the-heck-is-inclusive-design/) considered
* [ ] Rendering performance (initial load time & 60fps) and [perceived performance](http://blog.teamtreehouse.com/perceived-performance) considered
* [ ] Documentation created or updated
* [ ] Tested in [sandbox](https://github.com/facebookincubator/create-react-app#getting-started) if new component or breaking change

### How does this PR make you feel?

yay

### CODE REVIEW

**Comment by brentertz** [comment]
Triangles don't render as expected in IE/Edge - No fill, only lines.

screen shot 2017-11-08 at 6 49 50 am

**Comment by brentertz** [comment]
Triangles do not properly fill section backgrounds in IE 11.

screen shot 2017-11-08 at 6 54 42 am

screen shot 2017-11-08 at 6 55 17 am

File: src/website/app/Canvas.js [hunk 1]
```diff
@@ -17,58 +17,43 @@
...
+  backgroundSize: 'cover',
+  bottom: 0,
+  left: 0,
+  mixBlendMode: 'hard-light',
+  position: 'absolute',
+  right: 0,
+  top: 0
+});
 
 export default class Canvas extends Component<Props> {
```
Comment (kylegach) [inline src/website/app/Canvas.js]: This no longer needs to be a class.

File: src/website/app/Canvas.js [hunk 2]
```diff
@@ -17,58 +17,43 @@
...
-      // trianglesExist = true;
-    }
-  }
-
   render() {
-    const { className, triangles = true, ...restProps } = this.props;
+    const { className, ...restProps } = this.props;
     const classes = className ? className : '';
     return (
       <Root className={`canvas ${classes}`} {...restProps}>
```
Comment (kylegach) [inline src/website/app/Canvas.js]: Does any code still use the `canvas` className? If not, we can remove it, simply spread `{...props}`, and type props as `{}`.

File: src/website/app/Nav.js [hunk 3]
```diff
@@ -36,8 +36,6 @@ const styles = {
   },
   link: ({ theme }) => ({
     display: 'block',
-    // top & bottom: results of `getNormalizedValue(pxToEm(5), theme.fontSize_ui)`
```
Comment (kylegach) [inline src/website/app/Nav.js]: Bit confused why this was removed in this effort?

**Comment by kylegach** [comment]
This is such a **_massive_** improvement. 👏 🙌 🎉

**Pull request merged**